autogpt/config/ai_config.py

```diff
@@ -2,12 +2,11 @@
 """
 A module that contains the AIConfig class object that contains the configuration
 """
-from __future__ import annotations
 
-import os
-from typing import Type
+from pathlib import Path
+from typing import List, Optional, Type
 import yaml
-
+from __future__ import annotations
```

Comment:
This counts as sneaking in extra changes

Reply:
Not my change.  I was dealing with merge conflicts of people who wanted to use | in the __init__ types.

Reply:
It is in your PR, so it's your change. If it was like this in `master`, it wouldn't show up in the diff.